src/sidebar/components/tag-editor.js

```diff
@@ -17,9 +17,11 @@ let tagEditorIdCounter = 0;
 
 /**
  * @typedef TagEditorProps
- * @prop {(a: Object<'tags', string[]>) => any} onEditTags - Callback that saves the tag list.
- * @prop {string[]} tagList - The list of editable tags as strings.
- * @prop {Object} tags - Services
+ * @prop {(a: string) => boolean} onAddTag - Callback to add a tag to the annotation
```

Comment:
```suggestion
 * @prop {(tag: string) => boolean} onAddTag - Callback to add a tag to the annotation
```

If a descriptive name for the parameter is short, you might as well use it.